ksqldb-engine/src/test/java/io/confluent/ksql/services/SandboxedKafkaTopicClientTest.java

```diff
@@ -78,6 +78,8 @@ private SandboxedKafkaTopicClientTest() {
           .ignore("describeTopic", String.class)
           .ignore("describeTopics", Collection.class)
           .ignore("deleteTopics", Collection.class)
+          .ignore("listTopicsStartOffsets", Collection.class)
+          .ignore("listTopicsEndOffsets", Collection.class)
```

Comment:
Can you add tests to `SupportedMethods` for these new methods?

Reply:
ack